manifests/slave.pp

```diff
@@ -150,7 +150,7 @@
     ensure => 'file',
     mode   => '0755',
     owner  => 'root',
-    group  => 'root',
+    group  => '0',
```

Comment:
I have a vague memory of a similar change in another module causing problems but I don't recall which module.  I'll see if I can find a reference.